enterprise/kerberos-login-module/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>kerberos-login-module</groupId>
+    <artifactId>kerberos-login-module</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <name>Kerberos Login Module</name>
+    <description>A Kerberos Login Module for Hazelcast.</description>
+    
+    <!--
+    <packaging>jar</packaging>
+    -->
+
+    <parent>
+        <artifactId>enterprise</artifactId>
+        <groupId>com.hazelcast.samples.enterprise</groupId>
+        <version>0.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <properties>
+        <!-- needed for checkstyle/findbugs -->
+        <main.basedir>${project.parent.parent.basedir}</main.basedir>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    
+    <build>
+      <plugins>
+        <plugin>
+          <!--
+          <groupId></groupId>
+          -->
+          <artifactId>kerberos-login-module</artifactId>
+          <version>${hazelcast.version}</version>
+          <configuration>
+            <archive>
+              <manifestFile>src/main/resources/META-INF/MANIFEST.MF</manifestFile>
+            </archive>
+          </configuration>
+        </plugin>
+      </plugins>
+    </build>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast-all</artifactId>
+            <version>${hazelcast.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.hazelcast.samples</groupId>
+            <artifactId>helper</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        
+        <dependency>
+          <groupId>KerberosUtils</groupId>
+          <artifactId>KerberosUtils</artifactId>
+          <version>0.0</version>
+          <scope>system</scope>
+          <systemPath>${project.basedir}/lib/KerberosUtils.jar</systemPath>
+        </dependency>
```

Comment:
We should avoid system dependencies.

Reply:
Understood. I am using classe for [this library](https://github.com/jcmturner/java-kerberos-utils) which is published on GitHub but does not seem to be available on a Maven repository of some sort - what would be the best solution here? Copy & embed the code in our module? Or?

Reply:
Have experimented with JaasLounge and other libs - all libs are quite old, which is ... weird ... anyways the simplest solution for now seems to be to embed the bare minimum of the KerberosUtils lib I was using. So, have updated the PR.